HW2/HW2EX1/ses-test/cmd.ap.sub_03

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env tcsh
+
+# created by uber_subject.py: version 0.40 (March 30, 2017)
+# creation date: Thu Aug  3 22:18:44 2017
+
+# set data directories
+set top_dir   =                                                             \
+    /Users/timmichaels/Documents/UConn/BIRC/WedJuly26/Non-Linear/playground/HW2
+set anat_dir  = $top_dir/ses-test/anat
+set epi_dir   = $top_dir/ses-test/func
+
+# set subject and group identifiers
+set subj      = sub_03
+set group_id  = test
+
+# run afni_proc.py to create a single subject processing script
+afni_proc.py -subj_id $subj                                                 \
+        -script proc.$subj -scr_overwrite                                   \
+        -blocks tshift align tlrc volreg blur mask scale regress            \
+        -copy_anat $anat_dir/sub-03_ses-test_T1w.nii.gz                     \
+        -tcat_remove_first_trs 0                                            \
+        -dsets $epi_dir/sub-03_ses-test_task-linebisection_bold.nii.gz      \
+        -tlrc_base MNI_avg152T1+tlrc                                        \
+        -volreg_align_to third                                              \
+        -volreg_align_e2a                                                   \
+        -volreg_tlrc_warp                                                   \
+        -blur_size 6.0                                                      \
+        -regress_stim_times                                                 \
+            $top_dir/detection.1D                                           \
+            $top_dir/landmark.1D                                            \
+        -regress_stim_labels                                                \
+            detection landmark                                              \
+        -regress_basis 'GAM'                                                \
+        -regress_censor_motion 0.5                                          \
+        -regress_apply_mot_types demean deriv                               \
+        -regress_opts_3dD                                                   \
+            -gltsym 'SYM: detection -landmark' -glt_label 1 D-L             \
```

Comment:
Then your critical contrast will be landmark correct + landmark incorrect -2*detection